admin/admin.js

```diff
@@ -213,7 +213,7 @@ Meteor.methods({
   "ts-admin-account-balance"() {
     TurkServer.checkAdmin();
     try {
-      return TurkServer.mturk("GetAccountBalance", {});
+      return TurkServer.mturk.getAccountBalanceSync().AvailableBalance;
```

Comment:
Every `Turkserver.mturk` in the codebase now needs to be replaced with this direct JavaScript function variant instead.  Note in particular that `peerlibrary:aws-sdk` conveniently provides a `...Sync()` variant of the function that works with Meteor's fibers, so the call looks synchronous (even though it is in fact a callback under the hood).